orchestrator/db/database.py

```diff
@@ -220,6 +220,10 @@ def database_scope(self, **kwargs: Any) -> Generator["Database", None, None]:
         self.scoped_session.remove()
         self.request_context.reset(token)
 
+    def update(self, *args: Any, **kwargs: Any) -> None:
```

Comment:
I dont think this should be here. Even if its a noop

Reply:
I added it to remove the type: ignore

Reply:
I prefer the type ignore over this.. You could also cast the db thing where you call update to Wrapped again